src/webhint-theme/source/js/telemetry/telemetry.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
+(function () {
+    const activityKey = 'webhint-activity';
+    const productKey = 'webhint-online-scanner';
+    const storage = window.localStorage;
+    const telemetryApiEndpoint = 'https://webhint-telemetry-api-appservice.azurewebsites.net/api/webhint-telemetry-ingress-api';
```

Comment:
Could this submit to an endpoint served as part of our existing telemetry repo instead? https://github.com/webhintio/telemetry

That already contains Azure Functions which work with our Application Insights instance. I was hoping we could add a new function there for this endpoint that we can share between here and our extensions.

Reply:
https://github.com/webhintio/telemetry/pull/164